bin/codeclimate-ruby

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require "codeclimate-test-reporter"
+
+COVERAGE_FILE = "coverage/.resultset.json".freeze
+
+if ENV["CODECLIMATE_REPO_TOKEN"]
+  if File.exist?(COVERAGE_FILE)
+    result = JSON.parse(File.read(COVERAGE_FILE))
+
+    CodeClimate::TestReporter::Formatter.new.format(result)
```

Comment:
Should you do some rescues here to make the failure cases (JSON parse-error or the reporter blowing up) more readable than a bare backtrace?